test/Benchmarks/Exporter/OtlpGrpcExporterBenchmarks.cs

```diff
@@ -33,7 +35,7 @@ public void GlobalSetup()
         this.exporter = new OtlpTraceExporter(
             options,
             new SdkLimitOptions(),
-            new OtlpGrpcTraceExportClient(options, new TestTraceServiceClient()));
+            new OtlpExporterTransmissionHandler<ExportTraceServiceRequest>(new OtlpGrpcTraceExportClient(options, new TestTraceServiceClient())));
```

Comment:
I notice that this change is only in the GlobalSetup. It should not affect the outcome of the benchmarks. Out of curiosity, have you validated to ensure that nothing has changed here?

Reply:
Http

Main

| Method                | NumberOfBatches | NumberOfSpans | Mean        | Error     | StdDev    | Gen0        | Gen1        | Gen2       | Allocated  |
|---------------------- |---------------- |-------------- |------------:|----------:|----------:|------------:|------------:|-----------:|-----------:|
| OtlpExporter_Batching | 1               | 10000         |    32.01 ms |  0.622 ms |  0.691 ms |   1666.6667 |   1600.0000 |   466.6667 |   18.33 MB |
| OtlpExporter_Batching | 10              | 10000         |   339.30 ms |  6.702 ms |  8.714 ms |  18000.0000 |  17000.0000 |  4000.0000 |   183.3 MB |
| OtlpExporter_Batching | 100             | 10000         | 3,227.09 ms | 34.603 ms | 30.675 ms | 178000.0000 | 175000.0000 | 50000.0000 | 1833.03 MB |

Change

| Method                | NumberOfBatches | NumberOfSpans | Mean        | Error     | StdDev    | Gen0        | Gen1        | Gen2       | Allocated  |
|---------------------- |---------------- |-------------- |------------:|----------:|----------:|------------:|------------:|-----------:|-----------:|
| OtlpExporter_Batching | 1               | 10000         |    32.95 ms |  0.654 ms |  1.449 ms |   1666.6667 |   1600.0000 |   466.6667 |   18.33 MB |
| OtlpExporter_Batching | 10              | 10000         |   342.55 ms |  6.656 ms |  9.110 ms |  18000.0000 |  17000.0000 |  4000.0000 |   183.3 MB |
| OtlpExporter_Batching | 100             | 10000         | 3,213.14 ms | 62.976 ms | 61.851 ms | 178000.0000 | 175000.0000 | 50000.0000 | 1833.03 MB |

Grpc

Main

| Method                | NumberOfBatches | NumberOfSpans | Mean        | Error     | StdDev    | Gen0        | Gen1        | Gen2       | Allocated  |
|---------------------- |---------------- |-------------- |------------:|----------:|----------:|------------:|------------:|-----------:|-----------:|
| OtlpExporter_Batching | 1               | 10000         |    14.08 ms |  0.280 ms |  0.547 ms |   1718.7500 |   1687.5000 |   500.0000 |   18.31 MB |
| OtlpExporter_Batching | 10              | 10000         |   146.11 ms |  2.899 ms |  4.157 ms |  17750.0000 |  17250.0000 |  4750.0000 |  183.12 MB |
| OtlpExporter_Batching | 100             | 10000         | 1,386.83 ms | 12.075 ms | 10.083 ms | 176000.0000 | 173000.0000 | 50000.0000 | 1831.15 MB |

Change
| Method                | NumberOfBatches | NumberOfSpans | Mean        | Error     | StdDev    | Gen0        | Gen1        | Gen2       | Allocated  |
|---------------------- |---------------- |-------------- |------------:|----------:|----------:|------------:|------------:|-----------:|-----------:|
| OtlpExporter_Batching | 1               | 10000         |    14.46 ms |  0.285 ms |  0.435 ms |   1750.0000 |   1718.7500 |   500.0000 |   18.31 MB |
| OtlpExporter_Batching | 10              | 10000         |   146.44 ms |  2.891 ms |  4.415 ms |  17750.0000 |  17250.0000 |  4750.0000 |  183.12 MB |
| OtlpExporter_Batching | 100             | 10000         | 1,393.41 ms | 17.925 ms | 15.890 ms | 176000.0000 | 173000.0000 | 50000.0000 | 1831.15 MB |